nss/integration-tests/integration_test.go

```diff
@@ -148,6 +149,18 @@ func TestIntegration(t *testing.T) {
 			got, status := getentOutputForLib(t, libPath, socketPath, rustCovEnv, tc.shouldPreCheck, cmds...)
 			require.Equal(t, tc.wantStatus, status, "Expected status %d, but got %d", tc.wantStatus, status)
 
+			if tc.shouldPreCheck && tc.db == "passwd" {
```

Comment:
I think we've a const for `"passwd"` now, right?

Reply:
do you mean this const? https://github.com/ubuntu/authd/blob/2e0785122cb110f692c8b48d0d42b3a3fd1f2a59/internal/brokers/auth/consts.go#L24-L24

we have that const, but what we're checking here is if the testcase `db` field matches the "passwd" literal string which we set [above](https://github.com/ubuntu/authd/blob/e186ecad42451ca1aea45e00fae68263aa6cccde/nss/integration-tests/integration_test.go#L58-L58).

Reply:
Yeah, but it's not the one... Since that's for db only, so maybe we should add one, one day...